org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/dom/Node.java

```diff
@@ -351,11 +351,11 @@ public boolean isComment() {
 	}
 
 	public boolean isProcessingInstruction() {
-		return getNodeType() == Node.PROCESSING_INSTRUCTION_NODE;
+		return (getNodeType() == Node.PROCESSING_INSTRUCTION_OR_PROLOG_NODE) && ((ProcessingInstruction) this).isProcessingInstruction();

```

Comment:
Please keep my original code if you ar eOK with my constants PROCESSING_INSTRUCTION_NODE